openedx/core/djangoapps/credentials/api.py

```diff
@@ -0,0 +1,15 @@
+"""
+Python APIs exposed by the credentials app to other in-process apps.
+"""
+
+
+from openedx.core.djangoapps.credentials.models import CredentialsApiConfig
+
+
+def is_credentials_enabled():
```

Comment:
I plan on updating other places/files that import the `CredentialsApiConfig` model directly and call `is_learner_issuance_enabled`, but I didn't want to distract anymore from the (more) important changes in this PR. 

I will do this in a follow up PR.